orca-pipelinetemplate/src/test/resources/gate.yml

```diff
@@ -0,0 +1,150 @@
+#document netflix-conventions
+clusters:
+  netflix-conventions:
+    requires:
+      membership:
+        - nf-datacenter
+        - nf-infrastructure
+
+accounts:
+  awsdefault:
+    cloudProvider: aws
+    regions:
+      us-east-1:
+        name: us-east-1 #baerf
+        zones:
+          - us-east-1c
+          - us-east-1d
+          - us-east-1e
+      us-west-2:
+        name: us-west-2  #backrefs?
```

Comment:
One concern I have with the jackson route is that we're making changes to the graph (adding, removing) based on ID in stages & modules after deserialization. If we go the jackson route, won't we need to jump through some hoops to get ids assigned correctly to child objects?